### ISSUE TITLE
SR2 - Related Services links should be in a list

### ISSUE THREAD

**Issue by jpinsent** [brharp/hjckrrh#239]
Related Services links are not sorted into a list. They're just organized into divs.

![related services](https://cloud.githubusercontent.com/assets/12450480/10284511/5c1150fc-6b52-11e5-97e1-0184ee879f53.PNG)
